Assets/Voxelgon/Ship/ShipManager.cs

```diff
@@ -1,69 +1,77 @@
-﻿using UnityEngine;
-using System;
-using System.Collections;
-using System.Collections.Generic;
-using Voxelgon;
-
-[RequireComponent (typeof (Rigidbody))]
-
-public class ShipManager : MonoBehaviour {
-
-    public float portYawCutoff = 30;    //angle +/- before the port is no longer for rotation
-
-    //input Variables
-    public float linInput;
-    public float latInput;
-    public float yawInput;
-
-    public bool killRot;
-    public bool killTrans;
-
-    //translation/yaw
-    public float yaw;
-    public float lin;
-    public float lat;
-
-    public float brakingYaw = 0;
-    public float brakingLin = 0;
-    public float brakingLat = 0;
-
-    private void FixedUpdate() {
-        yawInput = Input.GetAxis("Yaw");
-        linInput = Input.GetAxis("Thrust");
-        latInput = Input.GetAxis("Strafe");
-
-        if(Input.GetButtonUp("Kill Rotation")) {
-            killRot = !killRot;
-        }
-
-        if(Input.GetButtonUp("Kill Translation")) {
-            killTrans = !killTrans;
+﻿using System;
+using UnityEngine;
+
+namespace Assets.Voxelgon.Ship
```

Comment:
This should really just be Voxelgon.Ship
Also no newline before opening brace, please